src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/Example5.txt

```diff
@@ -0,0 +1,16 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="WriteTag"/>
+  </module>
+</module>
+*/
+
+// xdoc section -- start
+class InputWriteTagNoJavadoc
+{
+    public void method()
+    {
```

Comment:
Please put `{` on same line with declaration of class and method.
It will make example compact, and it is most usual style.

Reply:
instead of writing new Example.

Please add `public void method() {}` without javadoc to all above Examples.

```
/**
* Some class
*/
public class Test {
  /** some doc */
  void foo() {}

  public void method() {}
}
  
```